doc/standard_library.md

```diff
@@ -0,0 +1,214 @@
+
+## Libraries
+
+- [benchmark]: Provides methods to measure and report the time used to execute code
+- [bundler]: Manage your Ruby application's gem dependencies
+- [cgi]: Support for the Common Gateway Interface protocol
+- [delegate]: Provides three abilities to delegate method calls to an object
+- [did_you_mean]: "Did you mean?" experience in Ruby
+- [English]: Provides references to special global variables with less cryptic names
+- [erb]: An easy-to-use but powerful templating system for Ruby
+- [error_highlight]: Highlight error locations in your code
+- [fileutils]: Several file utility methods for copying, moving, removing, etc.
+- [find]: This module supports top-down traversal of a set of file paths
+- [forwardable]: Provides delegation of specified methods to a designated object
+- [ipaddr]: Provides methods to manipulate IPv4 and IPv6 IP addresses
+- [irb]: Interactive Ruby command-line tool for REPL (Read Eval Print Loop)
+- [optparse]: Ruby-oriented class for command-line option analysis
+- [logger]: Provides a simple logging utility for outputting messages
+- [net-http]: HTTP client API for Ruby
+- [open3]: Provides access to stdin, stdout, and stderr when running other programs
+- [ostruct]: A class to build custom data structures, similar to a Hash
+- [open-uri]: An easy-to-use wrapper for URI::HTTP, URI::HTTPS, and URI::FTP
+- [pp]: Provides a PrettyPrinter for Ruby objects
+- [prettyprint]: Implements a pretty printing algorithm for readable structure
+- [prism]: A portable, error-tolerant Ruby parser
+- [pstore]: Implements a file-based persistence mechanism based on a Hash
+- [readline]: Wrapper for the Readline extension and Reline
+- [reline]: GNU Readline and Editline in a pure Ruby implementation
+- [resolv]: Thread-aware DNS resolver library in Ruby
+- [rdoc]: Produces HTML and command-line documentation for Ruby
+- [securerandom]: Interface for a secure random number generator
+- [set]: Provides a class to deal with collections of unordered, unique values
+- [shellwords]: Manipulates strings with the word parsing rules of the UNIX Bourne shell
+- [singleton]: Implementation of the Singleton pattern for Ruby
+- [tempfile]: A utility class for managing temporary files
+- [time]: Extends the Time class with methods for parsing and conversion
+- [timeout]: Auto-terminate potentially long-running operations in Ruby
+- [tmpdir]: Extends the Dir class to manage the OS temporary file path
+- [tsort]: Topological sorting using Tarjan's algorithm
+- [un]: Utilities to replace common UNIX commands
+- [uri]: A Ruby module providing support for Uniform Resource Identifiers
+- [yaml]: The Ruby client library for the Psych YAML implementation
+- [weakref]: Allows a referenced object to be garbage-collected
+
+## Extensions
+
+- [date]: A subclass of Object that includes the Comparable module for handling dates
+- `DateTime`: A subclass of Date for handling dates, hours, minutes, seconds, and offsets
```

Comment:
```suggestion
- [date]: Represents dates, with a subclass for dates with time and timezones.
```

I realize `DateTime` was listed separately previously, but I think that was a mistake.  We should only be listing the library here (`DateTime` is not a separate library, just a class provided by the `date` library).